### ISSUE TITLE
Attention-Mask

### ISSUE THREAD

**Issue by henry-AY** [henry-AY/Mini-TensorRT#1]
* The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
